src/parcsr_ls/par_mgr_solve.c

```diff
@@ -787,9 +802,11 @@ hypre_MGRCycle( void               *mgr_vdata,
                hypre_ParCSRMatrixMatvecOutOfPlace(-1.0, A_array[level],
                                                   U_array[level], 1.0, F_array[level], Vtemp);
 
-               resnorm = hypre_ParVectorInnerProd(Vtemp, Vtemp);
+               hypre_ParVectorInnerProd(Vtemp, Vtemp, &resnorm);
```

Comment:
@oseikuffuor1, this fixes a possible bug in MGR. Previously `resnorm = <Vtemp, Vtemp>`, while it makes sense to take the square root here. Could you please verify?